src/pages/login/__tests__/login.spec.ts

```diff
@@ -34,6 +37,11 @@ describe('LoginPage', () => {
     it('should create', () => {
       expect(component).toBeDefined();
     });
+    it('should login successfully', () => {
+      component.login().then(() => {
+        expect(navController.setRoot).toBeCalled;
```

Comment:
The call seems to be missing here, I don't think you're checking anything. Would be nice to test it's called with the correct argument too.

Reply:
Fixed